users/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db      import models
```

Comment:
외부 파이썬 패키지 impor문과 사용자 정의 모듈 사이에 한 칸 띄워주세요.
```suggestion

from django.db      import models
```